src/services/mongoose/wallets.ts

```diff
@@ -102,13 +103,28 @@ export const WalletsRepository = (): IWalletsRepository => {
       return new UnknownRepositoryError(err)
     }
   }
+  const listAllWalletIds = async (
+    walletCurrency: WalletCurrency,
+  ): Promise<WalletId[] | RepositoryError> => {
+    try {
+      const result: WalletRecord[] = await Wallet.find({ currency: walletCurrency })
```

Comment:
this cast is not needed anymore since medici 5 afaik

```suggestion
      const result = await Wallet.find({ currency: walletCurrency })
```